### ISSUE TITLE
Implement Institutional Partner Page BE

### ISSUE THREAD

**Issue by BruceP99166** [openstax/business-intel#277]
# Description
This card is for BE work

**What is being developed?**
A new web page for the Institutional Partner program.

**Who is the audience for this and what will they be able to do when it is released?**
The audience for this page consists of senior administrators at the institutional level, such as Chancellors, Presidents, Provosts, Vice Chancellors/Presidents of Academic Affairs; OER Coordinators; and Conference organizers. 

When this page is released, they will be able to get detailed information about the Institutional Partner program and its benefits, apply for the program when the application is open, and learn more about the resources OpenStax offers beyond our textbooks.

**Why do it?**
The Institutional Partner program has tremendous benefits for OpenStax. While it coaches institutions on how to adopt all kinds of OER, not just OpenStax, for their students, they are consistently some of our highest ranking schools in terms of adoptions and student savings. See latest IR report here. In addition, the program helps us develop strategic relationships at key institutions. 

**What are the contractual requirements, and/or potential opportunities, and/or risks to be averted?**
If we don’t continue to promote and expand the Institutional Partnership program, there is a risk that institutions will turn to Lumen Learning and other competitors for answers. We need somewhere for schools that are interested in increasing OER use to go for the answers and support they’re looking for. 

**When should it be released?**
Q1 2019

**Why should it be released then?**
We want institutions to have plenty of time to learn more about the program and determine if they should apply before the application opens in late spring 2019.

# Acceptance
 There is a page on OpenStax.org under the "What we do" section of the website.
The page is called "Institutional Partnerships"
The page allows a user to download a brochure about the program.
The page allows a user to sign up for more information about the program.
The page allows a user to download an OER Summer Action Plan checklist and links to the Open Oregon Webinar hosted by Nicole.
The page contains the logos of current participants in the program, and a link expands to show the past participant logos.
During the application season, the page has a button taking users to the Institutional Partnership application page.

# Epic Doc

https://docs.google.com/document/d/1P7lrNh7zSDXspphYA2AUyIMxXNdAB5tUkOxX6wus1XQ/edit#heading=h.wuc3pn1ckk21

# Checklist for Done
1A-DEFINE
- [ ] Description is complete 
- [ ] High-level natural language acceptance tests are complete
- [ ] If UX design is needed then the UX label is added
- [ ] If UX design is needed then the UX LoFi Design is completed, approved by the Product Manager, and added to the issue.
- [ ] A high-level solution (development approach for the code change) is added to the issue
- [ ] If prototype code is generated, it is linked to the issue.
- [ ] If tech briefs are generated, they are linked to the issue.

1B-DECOMP
- [ ] The work is broken into reasonable and consistent work items.
- [ ] Story issues are created for the work items, with Description, Acceptance, the label 'story', and an Estimate.
- [ ] The original issue is changed to an epic and the child story issues are linked to it.  The estimate is removed.
- [ ] The story epic and child work item issues are moved into 1B-DECOMP DONE.
- [ ] If an issue is included in a product phase, then Release is selected.

1C-DESIGN
  - [ ] The UX HiFi Design is complete, approved by the Product Manager, and added to the issue.
  - [ ] Distilled acceptance tests are added to the issue.
  - [ ] Automated testing criteria are added to issue.
  - [ ] The solution is updated.

2-CODE
  - [ ] A pull request is opened, the automated pull request checks pass, and the change has been approved by other developers.
  - [ ] The acceptance tests are finalized in "Given / When / Then" format.
  - [ ] Regression test categories are identified on the issue. Categories should be referenced if any changes might affect them, even if the intended functionality is unchanged.

4A-UX REVIEW
  - [ ] The pull request is merged into master, and the change branch is deleted.

5A-FUNCT VER
  - [ ] All the acceptance tests for this issue pass.

5C-REGRESSION
  - [ ] The test plan in testrail passes.
  - [ ] Blessing from Product Manager that the release is good to go.

5D-READY TO BE DEPLOYED
  - [ ] Production is updated and verified.
  - [ ] All issues in the release are closed.

**QA needs checklist:**
  - [ ] Acceptance Criteria are complete
  - [ ] Milestone should be tagged for PR's 
  - [ ] Milestone should be tagged for issues
  - [ ] Content update needed
  - [ ] Content update complete

**Deployment needs**
  - [ ] Pre-deployment tasks
  - [ ] Data migrations 
  - [ ] Post-deployment tasks
  
_If this issue becomes an epic, delete from 1C-DESIGN to here; otherwise, delete from here to the end_

TRACK / EPICS
- [ ] All child issues are moved to 3B-RELEASE REVIEW.

3B-RELEASE REVIEW
- [ ] Acceptance tests pass for all issues in the release
- [ ] Blessing from Product Manager that the release is good to go.

3C-RELEASED
- [ ] Production is updated and verified.
- [ ] All issues in the release are closed.

**Comment by RoyEJohnson** [comment]
@michael This is what I'm using as mock data for now
```
        this.pageData = {
            section_1_heading: 'Institutional Partner Program',
            section_1_description: `
            Apply now for the 2019-2020 program! Applications are due <b>April 30, 2019</b>.`,
            section_1_link_text: 'Apply now',
            section_1_link: 'http://example.com/apply',
            quote: `Having to develop our strategic plan early on and having monthly
            conversations with our equally engaged collagues from other schools was
            very motivating and the jump start that carried us to new heights.`,
            quote_name: 'Kathy Labadorf',
            quote_title: 'Open Educational Resources, Information Literacy, and WGSS Librarian',
            quote_school: 'University of Connecticut',
            section_2_heading: 'About the program',
            section_2_description: `
            <p>Rice Univerity's OpenStax is one of the leading publishers of open
            educational resources (OER) in the world. With a library of over 30 free,
            high-quality, peer-reviewed textbooks, Openstax is breaking down the most
            common barriers to learning</p><p>That's why we created the Openstax
            Institutional Partner Program, which is designed to provide institutions
            with guidance to greatly increase the use of open educational resources
            on their campuses through a structured strategic plan and community of
            peers. The program teaches institutions the most effective strategies for
            encouraging faculty adoption of OER in general, not just of OpenStax.</p>
            <p>This program is for institutions ready to make an impact on campus by
            dedicating teh necessary time and resources to motivate faculty and
            utilize open educational resources to drive student success, retention,
            and completion.</p>
            <p>Download <a href="http://example.com/brochure">this brochure</a> for
            more details about the program.</p>`,
            section_2_image: 'http://via.placeholder.com/440x470/998866',
            section_2_image_alt: 'A view of Rice',
            section_3_heading: 'Promoting OER on your campus',
            section_3_description: `Institutions don't have to be in the Institutional
            Partnership Program to start encouraging the use of OER on campus. Below
            are a few tips to get started.`,
            section_3_wide_cards: [
                {
                    icon: 'http://via.placeholder.com/48/4488FF',
                    html: `<b>Create some goals</b> focusing on outcomes and put together
                    a plan to reach those OER goals`
                },
                {
                    icon: 'http://via.placeholder.com/48/4488FF',
                    html: `<b>Plan a series of events with faculty promoting OER on
                    your campus`
                },
                {
                    icon: 'http://via.placeholder.com/48/4488FF',
                    html: `<b>Follow up with faculty</b> attendees to see if they're
                    interested in piloting or adopting OER`
                }
            ],
            section_3_tall_cards: [
                {
                    html: `For more tips on how to get started, download our
                    <b>Summer Action Plan Checklist</b>.`,
                    link: 'http://example.com/download',
                    link_text: 'Download'
                },
                {
                    html: `Check out this <b>Open Oregon webinar</b> on effective
                    strategies to increase OER adoptions on your campus, hosted by
                    Nicole Finkbeiner, director of institutional relations at OpenStax.`,
                    link: 'http://example.com/download',
                    link_text: 'Download'
                }
            ],
            section_4_quote_text: `Our partnership with OpenStax fostered lifelong
            connections that benefit not only OER collaborations, but student success
            between institutions.`,
            section_4_quote_name: 'Brian Weston',
            section_4_quote_title: 'Director of Distance and Accelerated Learning',
            section_4_quote_school: 'College of the Canyons',
            section_5_heading: 'Speaking engagements & webinars',
            section_5_description: `
            <p>After five years of running the Institutional Partnership Program,
            Nicole Finkbeiner blah blah blah</p>
            Book Nicole for your next campus event by contacting
            <a href="mailto:something@rice.edu">institutionalpartners@openstax.org</a>`,
            section_5_image: 'http://via.placeholder.com/440x470/665533',
            section_5_image_alt: 'Nicole at lectern',
            section_6_heading: 'Big results',
            section_6_description: `Last year, institutions in the program, on average,
            dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
            ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
            exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis
            aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. `,
            section_6_cards: [
                {
                    heading_number: '161',
                    heading_unit: '%',
                    description: 'Students use OpenStax textbooks'
                },
                {
                    heading_number: '17.4',
                    heading_unit: 'Million',
                    description: 'projected student savings this year'
                },
                {
                    heading_number: '150',
                    heading_unit: '%',
                    description: 'more students impacted by OER this year'
                },
                {
                    heading_number: '4.9',
                    heading_unit: 'Million',
                    description: 'in additional student savings'
                }
            ],
            section_7_heading: 'Program Participants',
            section_7_subheading: '2019-2020 Cohort',
            section_7_icons: [
                {
                    image: 'http://via.placeholder.com/180x64?text=school%20name',
                    image_alt_text: 'school name'
                },
                {
                    image: 'http://via.placeholder.com/180x64?text=school%20name',
                    image_alt_text: 'school name'
                },
                {
                    image: 'http://via.placeholder.com/120x110?text=school%20name',
                    image_alt_text: 'school name'
                },
                {
                    image: 'http://via.placeholder.com/180x64?text=school%20name',
                    image_alt_text: 'school name'
                },
                {
                    image: 'http://via.placeholder.com/180x64?text=school%20name',
                    image_alt_text: 'school name'
                },
                {
                    image: 'http://via.placeholder.com/180x64?text=school%20name',
                    image_alt_text: 'school name'
                },
                {
                    image: 'http://via.placeholder.com/180x64?text=school%20name',
                    image_alt_text: 'school name'
                }
            ],
            section_7_link_text: 'See established partners',
            section_7_link_target: '/partners',
            section_8_quote_text: `Our partnership with OpenStax fostered lifelong
            connections that benefit not only OER collaborations, but student success
            between institutions.`,
            section_8_quote_name: 'Brian Weston',
            section_8_quote_title: 'Director of Distance and Accelerated Learning',
            section_8_quote_school: 'College of the Canyons',
            section_9_heading: 'Sign up for updates',
            section_9_submit_url: 'http://example.com/formSubmit',
            section_9_form_prompt: 'Enter your email here.',
            section_9_button_text: 'Sign up',
            section_9_contact_html: `Have questions about the program? Please contact
            <br>
            <a href="mailto: institutionalpartners@openstax.org">institutionalpartners@openstax.org</a>`

        };
```

**Comment by RoyEJohnson** [comment]
This is the desktop design
https://projects.invisionapp.com/share/98QRRWRSVUN#/screens
and mobile design
https://projects.invisionapp.com/share/98QRRWRSVUN#/screens/353138984

**Comment by Brooke-Fail** [comment]
Waiting for the FE card to be completed to fully: test:https://app.zenhub.com/workspaces/openstax-bit-5bacfe3f4b5806bc2bea3764/issues/openstax/business-intel/276

Tested the Institutional Partner page in QA and everything seemed good. Also looked at the mock ups (web and mobile) and they seemed to fit the acceptance criteria.

**Comment by Brooke-Fail** [comment]
Verified in staging for the Hotfix. Will do more testing on this once the FE is ready.

**Comment by Brooke-Fail** [comment]
Verified the content does not show up in Production. Will close this issue when FE testing has completed.

**Comment by itsgeraldwang** [comment]
Verified in Production.
